packages/block-library/src/navigation-link/index.php

```diff
@@ -322,13 +322,33 @@ function build_variation_for_navigation_link( $entity, $kind ) {
 	return $variation;
 }
 
+/**
+ * Register a variation for a post type / taxonomy for the navigation link block
+ *
+ * @param array $variation Variation array from build_variation_for_navigation_link.
+ * @return void
+ */
+function register_block_core_navigation_link_variation( $variation ) {
```

Comment:
```suggestion
function gutenberg_register_block_core_navigation_link_variation( $variation ) {
```

As per https://github.com/WordPress/gutenberg/blob/trunk/lib/README.md. Let's wait for confirmation before updating.

Reply:
Ignore I was incorrect.